WordPress/Sniffs/WP/CapabilitiesSniff.php

```diff
@@ -0,0 +1,470 @@
+		'install_languages'           => true,
+		'install_plugins'             => true,
+		'install_themes'              => true,
+		'list_app_passwords'          => true,
+		'list_users'                  => true,
+		'manage_categories'           => true,
+		'manage_links'                => true,
+		'manage_network'              => true,
+		'manage_network_options'      => true,
+		'manage_network_plugins'      => true,
+		'manage_network_themes'       => true,
+		'manage_network_users'        => true,
+		'manage_options'              => true,
+		'manage_post_tags'            => true,
+		'manage_privacy_options'      => true,
+		'manage_sites'                => true,
+		'moderate_comments'           => true,
+		'publish_blocks'              => true, // Alias for 'publish_posts', but supported.
+		'publish_pages'               => true,
+		'publish_post'                => true, // Alias, but supported.
+		'publish_posts'               => true,
+		'promote_user'                => true,
+		'promote_users'               => true,
+		'read'                        => true,
+		'read_block'                  => true, // Only seen in tests.
+		'read_post'                   => true, // Alias, but supported.
+		'read_page'                   => true, // Alias, but supported.
+		'read_app_password'           => true,
+		'read_private_blocks'         => true, // Alias for 'read_private_posts', but supported.
+		'read_private_pages'          => true,
+		'read_private_posts'          => true,
+		'remove_user'                 => true, // Alias for 'remove_users', but supported.
+		'remove_users'                => true,
+		'resume_plugin'               => true, // Alias for 'resume_plugins', but supported.
+		'resume_plugins'              => true,
+		'resume_theme'                => true, // Alias for 'resume_themes', but supported.
+		'resume_themes'               => true,
+		'setup_network'               => true,
+		'switch_themes'               => true,
+		'unfiltered_html'             => true,
+		'unfiltered_upload'           => true,
+		'update_core'                 => true,
+		'update_https'                => true,
+		'update_languages'            => true,
+		'update_plugins'              => true,
+		'update_php'                  => true,
+		'update_themes'               => true,
+		'upgrade_network'             => true,
+		'upload_files'                => true,
+		'upload_plugins'              => true,
+		'upload_themes'               => true,
+		'view_site_health_checks'     => true,
+	);
+
+	/**
+	 * List of deprecated core capabilities.
+	 *
+	 * User Levels were deprecated in version 3.0.
+	 *
+	 * {@internal To be updated after every major release. Last updated for WordPress 6.1.0.}
+	 *
+	 * @link https://github.com/WordPress/wordpress-develop/blob/master/tests/phpunit/tests/user/capabilities.php
+	 *
+	 * @since 3.0.0
+	 *
+	 * @var array<string, string> All deprecated capabilities in core.
+	 */
+	protected $deprecated_capabilities = array(
```

Comment:
Any particular reason why this property is `protected` ? 


```suggestion
	private $deprecated_capabilities = array(
```

Reply:
No reason, this is left over from the original PR 👍🏼